### PR TITLE
[5.10.x]Change folder structure to support jdk8 and jdk11 docker images and change docker images accordingly.

### PR DESCRIPTION
**Pull request by bhagyasakalanka**
Related Issues: #306

### CODE REVIEW

File: dockerfiles/jdk8/alpine/is/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```
Comment (RidmiR) [inline dockerfiles/jdk8/alpine/is/Dockerfile:3]: Change the Licence Header

File: dockerfiles/jdk8/alpine/is/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```
Comment (RidmiR) [inline dockerfiles/jdk8/alpine/is/docker-entrypoint.sh:3]: Change the Licence Header

File: dockerfiles/jdk8/alpine/is/Dockerfile (hunk 1 above)
Comment (bhagyasakalanka) [reply dockerfiles/jdk8/alpine/is/Dockerfile]: fixed

File: dockerfiles/jdk8/alpine/is/docker-entrypoint.sh (hunk 2 above)
Comment (bhagyasakalanka) [reply dockerfiles/jdk8/alpine/is/docker-entrypoint.sh]: fixed

**Review by RidmiR — APPROVED** [review]

**Pull request merged**